Add LHR06 and LHR07

robertodauria opened the pull request:
This PR adds two new sites to siteinfo:
- LHR06: https://github.com/m-lab/ops-tracker/issues/927
- LHR07: https://github.com/m-lab/ops-tracker/issues/928

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/m-lab/siteinfo/79)
<!-- Reviewable:end -->

mmillner commented:
it is TI, they need to advertise it via there ASN. It is allocated in RADB
(I think that is the tool). Same scenario as the Tata site in FRA..google
IP's..what did you do for that one?

On Fri, Nov 15, 2019 at 9:07 AM nkinkade <notifications@github.com> wrote:

> *@nkinkade* requested changes on this pull request.
>
> Reviewed 2 of 3 files at r1.
> *Reviewable <https://reviewable.io/reviews/m-lab/siteinfo/79>* status: 1
> change requests, 0 of 1 approvals obtained (waiting on @nkinkade
> <https://github.com/nkinkade> and @robertodauria
> <https://github.com/robertodauria>)
>
> ------------------------------
>
> *sites/lhr07.jsonnet, line 17 at r1
> <https://reviewable.io/reviews/m-lab/siteinfo/79#-Ltjnw6I7jsw1r0eEiWc:-Ltjnw6I7jsw1r0eEiWd:b-zgeh7c>
> (raw file
> <https://github.com/m-lab/siteinfo/blob/e1909d9d4ab52c53de9192316b7207c18d550083/sites/lhr07.jsonnet#L17>):*
>
>   },
>   transit+: {
>     provider: 'TELECOM ITALIA SPARKLE S.p.A.',
>
> I saw the conversation in #ops-deployment about this with @mmillner
> <https://github.com/mmillner>. Is this the right provider? And do we want
> to leave the ASN blank? Is it just that the information has not been
> propagated in whois or otherwise? I don't know the implications of Google
> donating IP space to TI and how that affects ASN in this case. I don't know
> enought about this and would have to defer to someone else.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/m-lab/siteinfo/pull/79?email_source=notifications&email_token=AGMT5V34VMPCGSMAAUFTYRLQT3CMHA5CNFSM4JN34WU2YY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCLX7OZY#pullrequestreview-317716327>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AGMT5V7ZGYWWBVJ6F64V4UDQT3CMHANCNFSM4JN34WUQ>
> .
>


-- 
Mike Millner
Google
mmillner@google.com
Desk: 303-729-8192
Cell: *720-949-3462*

mmillner commented:
for TI I think it's 6762..should be the same as the TI site in Sao Paulo

On Fri, Nov 15, 2019 at 9:19 AM Stephen Soltesz <notifications@github.com>
wrote:

> *@stephen-soltesz* commented on this pull request.
>
> *Reviewable <https://reviewable.io/reviews/m-lab/siteinfo/79>* status: 1
> change requests, 0 of 1 approvals obtained (waiting on @mmillner
> <https://github.com/mmillner>)
>
> ------------------------------
>
> *sites/lhr07.jsonnet, line 17 at r1
> <https://reviewable.io/reviews/m-lab/siteinfo/79#-Ltjnw6I7jsw1r0eEiWc:-LtjqqW88kabYfK1npg-:b-4q8xnz>
> (raw file
> <https://github.com/m-lab/siteinfo/blob/e1909d9d4ab52c53de9192316b7207c18d550083/sites/lhr07.jsonnet#L17>):*
> *Previously, nkinkade wrote…*
>
> I saw the conversation in #ops-deployment about this with @mmillner
> <https://github.com/mmillner>. Is this the right provider? And do we want
> to leave the ASN blank? Is it just that the information has not been
> propagated in whois or otherwise? I don't know the implications of Google
> donating IP space to TI and how that affects ASN in this case. I don't know
> enought about this and would have to defer to someone else.
>
> The ASN should be the network that carries the server traffic irrespective
> of the providence of the server IPs. If the server IPs are ambiguous, what
> is the ASN of the first hop?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/m-lab/siteinfo/pull/79?email_source=notifications&email_token=AGMT5V2ZVVARWXRWQJU3AKDQT3DXVA5CNFSM4JN34WU2YY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCLYBI6Y#pullrequestreview-317723771>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AGMT5VZ3ZON3N7YXAEAV4YLQT3DXVANCNFSM4JN34WUQ>
> .
>


-- 
Mike Millner
Google
mmillner@google.com
Desk: 303-729-8192
Cell: *720-949-3462*

robertodauria commented:
I've checked the ASN on RADB and updated the asn field accordingly.

```route:      162.213.96.0/23
descr:      Google
origin:     AS6762
notify:     radb-contact@google.com
mnt-by:     MAINT-AS15169
changed:    radb-contact@google.com 20191010
source:     RADB```
